.nvmrc

```diff
@@ -1 +1 @@
-10.19.0
\ No newline at end of file
+20.0.0
```

Comment:
Make this just  20 in case the person working on it has a different 20.x.x version on their machine.